commands/container-registry/repository/repository.go

```diff
@@ -0,0 +1,64 @@
+package repository
+
+import (
+	"context"
+	"github.com/ionos-cloud/ionosctl/v6/commands/container-registry/registry"
+	"github.com/ionos-cloud/ionosctl/v6/pkg/core"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func RegRepoDeleteCmd() *core.Command {
+	cmd := core.NewCommand(
+		context.TODO(), nil, core.CommandBuilder{
+			Namespace: "container-registry",
+			Resource:  "repository",
```

Comment:
it's not clear from the command name that this command is a DELETE operation. Yes, the aliases and the description point out that in fact that is deleting the repo contents, but for that you need to actually type out the command.

As a solution, either rename to `repository-delete`, or maybe add a subcommand: `repository delete`? 

Note;   If we use `repository-delete`. we would have to make a breaking change if some other subcommand is added to `repo` namespace